picard/file.py

```diff
@@ -685,28 +688,28 @@ def clear_lookup_task(self):
     def set_pending(self):
         if self.state != File.REMOVED:
             self.state = File.PENDING
-            self.update()
+            self.update()  # FIXME: only call self.item.update() ??
```

Comment:
Yes, likely only `self.item.update()` is necessary for pending state changes. I'm pretty sure this update call is only abot updating the icon. I don't see what other change should happen that requires a call to `update()` that not already handles this itself.

Reply:
Done in https://github.com/metabrainz/picard/pull/1135/commits/1accac01df4dd7988b11014c1a7edb31e479a929